### PR TITLE
CA-203433: Raise an appropriate API error when we can't SXM a suspend…

### PR DESCRIPTION
**Pull request by jonludlam**
… image

Signed-off-by: Jon Ludlam <jonathan.ludlam@citrix.com>

### CODE REVIEW

**Comment by jonludlam** [comment]
Untested yet, so don't merge.

File: ocaml/xapi/xapi_vm_migrate.ml [hunk 1]
```diff
@@ -723,6 +723,13 @@ let migrate_send'  ~__context ~vm ~dest ~live ~vdi_map ~vif_map ~options =
          else acc)
       [] vm_and_snapshots in
 
+  (* Double check that all of the suspend VDIs are all visible on the source *)
```
Comment (simonjbeaumont) [inline ocaml/xapi/xapi_vm_migrate.ml]: should this comment say 'visible on the destination'?

Comment (robhoes) [inline ocaml/xapi/xapi_vm_migrate.ml]: It's checking on the source, before the migration. Which makes me think that this check should be done in `assert_can_migrate`...?

**Comment by robhoes** [comment]
@jonludlam What's up with this one?

**Comment by simonjbeaumont** [comment]
Closing, can always be re-opened.

**Pull request closed without merging**